vm/systemSmartContracts/auction.go

```diff
@@ -522,6 +522,7 @@ func (s *stakingAuctionSC) getVerifiedBLSKeysFromArgs(txPubKey []byte, args [][]
 		signedMessage := args[i+1]
 		err := s.sigVerifier.Verify(txPubKey, signedMessage, blsKey)
 		if err != nil {
+			s.eei.AddReturnMessage("invalid BLS key: " + err.Error())
```

Comment:
you need to define the bls key. and add message that money can be claimed back.

Reply:
done.